libs/langchain/langchain/llms/openai.py

```diff
@@ -731,12 +811,21 @@ class OpenAIChat(BaseLLM):
     """
 
     client: Any  #: :meta private:
+
+    # this is for compatibility with Union types in helper functions
+    async_client: Any  #: :meta private:
```

Comment:
OpenAIChat isn't actually compatible with v1. Just updated some easy things in case someone adds in future. Deprecated class